### PR TITLE
Chem dispenser rework 2

### PR DESCRIPTION
**Pull request by gozulio**
## About The Pull Request
Only includes the Chemical Dispenser changes from #836
Chem dispensers now take cartridges in order to produce reagents.

There are four variants of cartridges to load into your dispensers. You can either find them in the trash, craft them, or get them from medical vendors.
They are (in order from left to right)
Imitation chemical cartridge: Cheap hand crafted version.
Ancient chemical cartridge: Really bad version found in junk piles.
Salvaged chemical cartridge: decent version found for sale in vendors
Pristine chemical cartridge: highest end variant either bought or found in dungeons.

image

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
tweak: chemical dispensers now require chemical cartridges in order to function.
tweak: chem cartridges capacity doubled from their original values.
/:cl:

### CODE REVIEW

**Comment by gozulio** [comment]
PR should be ready. Wait until Sunday evening to actually merge.

**Review by Tk420634 — APPROVED** [review]

**Pull request merged**
